src/superadmin.yml

```diff
@@ -8,6 +8,7 @@ admins:
   f9a1982e-252e-4ed3-92ed-52b0506a39c9:
     last_login_name: Prozza
     is_activated: true
+    cmdspy_is_on: false
```

Comment:
As @JeromSar suggested it should  be on by default so instead of having cmdspy is on on you should make it cmdspy is off in the configuration